dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -28,6 +28,20 @@ If the titles of any of the below analyzers change, ensure [HelpLinkUrlFactory](
 
 To run VS experimentally set the `.Vsix` project as the startup project, which launches visual studio with the analysers installed. Depending on your use case create or open an existing project. Code that violates any analysers in this project should then be flagged.
 
+## Updating the analyzers to support a new C# version
+If the Microsoft.CodeAnalysis.CSharp.Workspaces package is being upgraded to support a new C# version (as per the official [Roslyn NuGet-packages.md](https://github.com/dotnet/roslyn/blob/main/docs/wiki/NuGet-packages.md)), the major version of Audacia.CodeAnalysis.Analyzers must be incremented.
+
+The Description in the `.csproj` must be updated to include the minimum version of C#/.NET that is supported.
+
+e.g
+
+```csharp
```

Comment:
If you format this as xml it won't highlight the integers and as operator

Reply:
tbh the code is in C# and is just an example so I'm happy to leave it as csharp myself rather than introduce potential confusion. If you strongly want me to change I will but imo I prefer it this way. Let me know though as you're obv more senior than me :)

Reply:
`.csproj` files are written in MSBuild [which is an XML schema ](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-project-file-schema-reference?view=vs-2022).

i.e.
```csharp
    <PropertyGroup>
        <Description>This package supports C# 12 and .NET 8 as a minimum.</Description>
    </PropertyGroup>
```

compared to
```xml
    <PropertyGroup>
        <Description>This package supports C# 12 and .NET 8 as a minimum.</Description>
    </PropertyGroup>
```

Reply:
Thanks Tom! I have now updated 😊